backend/src/main/java/codezap/template/controller/SpringDocTemplateController.java

```diff
@@ -55,25 +55,24 @@ public interface SpringDocTemplateController {
     })
     ResponseEntity<Void> createTemplate(MemberDto memberDto, CreateTemplateRequest createTemplateRequest);
 
-    @SecurityRequirement(name = "쿠키 인증 토큰")
-    @Operation(summary = "템플릿 조회", description = """
+    @Operation(summary = "템플릿 조회 (비회원)", description = """
             조건에 맞는 모든 템플릿을 조회합니다. \n
             - 조건 \n
               - 멤버 ID
               - 검색 키워드 (템플릿명, 템플릿 설명, 파일명, 소스 코드)
               - 카테고리 ID
               - 태그 ID들 \n
-              
+            
             페이징 조건을 줄 수 있습니다. 페이지 번호는 1, 템플릿 개수는 20, 정렬 방식은 최신순이 기본 값입니다. \n
             - 페이징 조건 \n
               - 페이지 번호(pageNumber)
               - 한 페이지에 템플릿 개수(pageSize)
               - 페이지 정렬 방식(sort) \n
-              
+            
             - 정렬 방식 \n
               - 최신순 (modifiedAt,asc)
-              - 오래된순 (modifiedAt,desc) 
-              - 좋아요 수가 많은 순 (like, desc) \n
+              - 오래된순 (modifiedAt,desc)
+              - 좋아요 수가 많은 순 (likesCount, desc) \n
```

Comment:
오늘 얘기가 나온 김에 리뷰 달아놓습니다~~ 나중에 논의를 해봐야할까요? 아니면 다른 의견 있으신 분은 댓글로? 어떻게 하는게 좋으려나~
```suggestion
              - 좋아요 순 (likesCount, desc) \n
```

Reply:
저는 이 수정방향 좋은데요~ 
좋아요순 이라는 선택지만 있으면 아무도 좋아요가 적은 순으로 정렬할 것이라 생각하지 않을 것 같아요~!